docs/i18n.rst

```diff
@@ -148,16 +131,6 @@ Once you've reviewed the changes, submit them in a pull request for the
 ``l10n: update strings to be translated for vX.Y.Z`` pull request as a
 template.
```

Comment:
This paragraph can be removed: source strings will be updated in the same branch (if not the very same commits) in which the enclosing source code is changed.